### ISSUE TITLE
Dialog: not possible to scroll content with arrow keys

### ISSUE THREAD

**Issue by hvinh** [angular/components#19479]
## Bug, enhancement request, or proposal

Bug

# Demo and steps to reproduce the issue

**Demo URL (required)***:  https://stackblitz.com/edit/angular-13nutg?file=src/app/dialog-overview-example-dialog.html

#### Detailed Reproduction Steps
1. Add more content in the dialog to make content bigger than the viewport
2. Try to scroll the content using up and down arrows

### Explain the expected behavior
The content of the dialog is scrolled

### Explain the current behavior
Content of the dialog is not scrolled

### Discuss the use-case or motivation for changing the existing behavior
This causes an issue for accessibility as keyboard only users cannot access the full content by scrolling

### List the affected versions of AngularJS, Material, OS, and browsers
- Angular Material 8.2.3
- Browsers: Chrome 81.0.4044.138

#### Screenshots
![image](https://user-images.githubusercontent.com/5523926/83165261-8b2f4880-a10d-11ea-9a59-5d044dba713c.png)

**Comment by mmalerba** [comment]
Here's a stackblitz repro: https://stackblitz.com/edit/angular-13nutg?file=src/app/dialog-overview-example-dialog.html

**Comment by hvinh** [comment]
Hello, thanks for your inputs I modified accordingly

**Comment by crisbeto** [comment]
In order for keyboard scrolling to work inside an element, the element has to contain focus. Your example doesn't work, because native inputs prevent the default up/down arrow key while they're focused. [Here's an example](https://stackblitz.com/edit/angular-13nutg-ybyfdb) that shows it working after you click on a button inside the content element. An alternative, if you want it to scrolling with the arrow keys when the user clicks on it, is to set `tabindex="-1"` on the `mat-dialog-content`.

I'm closing the issue since this is just how browsers work and there isn't much we can do about it.
